corehq/apps/case_search/utils.py

```diff
@@ -54,11 +54,15 @@ def _get_initial_search_es(self):
         return search_es
 
     def _assemble_optional_search_params(self):
+        self._add_include_closed()
         self._add_xpath_query()
         self._add_owner_id()
         self._add_blacklisted_owner_ids()
         self._add_case_property_queries()
 
+    def _add_include_closed(self):
+        self.search_es = self.search_es.is_closed(False)
```

Comment:
does it make sense to add this to `_get_initial_search_es` instead? Otherwise this function is a bit misleadingly named - I would have called it `remove_closed`.

Reply:
Yes, moving it would be much nicer - 5e3e72a5f18